src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,115 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exceptions.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationService REGISTRATION_SERVICE = new RegistrationServiceImpl();
+
+    @Test
+    void register_checkOnUserRegistration_isOk() {
+        User actual = new User();
+        actual.setLogin("Snowman");
+        actual.setPassword("123456");
+        actual.setAge(44);
+        User expected = REGISTRATION_SERVICE.register(actual);
+        assertEquals(expected, actual);
+    }
```

Comment:
The test case `register_checkOnUserRegistration_isOk` does not cover all the edge cases. You should add tests for the edge cases, such as a user with an age of 18 (minimum acceptable age), a password of 6 characters (minimum acceptable password length), and a login of 1 character (minimum acceptable login length). Remember, it's important to test the boundaries.